editor.planx.uk/src/@planx/components/Send/Editor.tsx

```diff
@@ -143,27 +134,29 @@ const SendComponent: React.FC<Props> = (props) => {
                     <ChecklistItem
                       id={option.value}
                       label={option.label}
+                      description={option.description}
                       onChange={changeCheckbox(option.value)}
                       checked={formik.values.destinations.includes(
                         option.value,
                       )}
-                      disabled={props.disabled}
+                      disabled={option.disabled || props.disabled}
                     />
                   </Grid>
                 ))}
               </Grid>
             </ErrorWrapper>
           </Box>
-          <Box display="flex" mt={2}>
-            <Warning titleAccess="Warning" color="primary" fontSize="large" />
-            <Typography variant="body2" ml={1}>
-              API tokens may be required to submit successfully. Check in with
-              PlanX developers before launching your service to make sure that
-              submissions are configured for your environment.
+          <WarningContainer>
+            <FactCheckIcon />
+            <Typography variant="body2" ml={2}>
+              View submissions activity for this service directly in the Plan✕
+              editor from the left menu. Successful submissions received less
+              than 28 days ago will be available to download by team editors.
             </Typography>
-          </Box>
+          </WarningContainer>
         </ModalSectionContent>
       </ModalSection>
+      <ModalFooter formik={formik} showMoreInformation={false} />
```

Comment:
The Send component previously didn't display tags or internal notes - these will be important for templates implementation !